### DISCUSSION TITLE
Import .ang file from EDAX, reference frame alignment

### DISCUSSION THREAD

**Discussion by fakhta15** [mtex-toolbox/mtex#1562] [Ask Anything]
Hi to all, 

I am quite new to using MTEX to explore my EBSD data. The data is acquired and imported to the .ang file with OIM EDAX software. When I run the import script for EBSD data, I got this message: 

> Warning: You have choosen to correct your EBSD data for differently aligned reference frames for the Euler angles and the map
> coordinates. However, you have not specified which reference system setting has been used on your Edax system . I'm going to assume
> "setting 1". Be careful, the default setting of EDAX is "setting 2". Click here for more information.Please make sure you have chosen
> the correct setting and specify it explicitely using the syntax
> 
>  ebsd = EBSD.load(fileName,'convertEuler2SpatialReferenceFrame','setting 2') 
> > In loadEBSD_ang (line 219)
> In EBSD.load (line 86)
> In Ni17_0N_1 (line 29)
> 
> the file name is : Ni17_0N_1.ang
> The path is D:\Users\mhatta\Documents\MATLAB\mtex-5.8.1\mtex-5.8.1\Ni17
> 

I found the instruction on how to do the reference frame alignment  and I follow these instructions: 

> ebsd = EBSD.load([mtexEBSDPath filesep 'Ni17_0N_1.ang'],'convertEuler2SpatialReferenceFrame','setting 2')
> Error using assert
> No file found!
> 
> Error in getFileNames (line 14)
>   assert(~isempty(files),'No file found!');

Could anyone please help me how I can handle this situation? 

THank you before, 
Fakhry

**Comment by lfgmorales** [comment]
Hi

You need to check on your EBSD data acquisition software (OIM DC, Team or Apex) which setting is used for the measurements, there are 4 possibilities. After you find out, you write ‘setting x’, where x is the setting number for measurements described above

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 16 Sep 2022, at 10:57, fakhta15 ***@***.***> wrote:

﻿

Hi to all,

I am quite new to using MTEX to explore my EBSD data. The data is acquired and imported to the .ang file with OIM EDAX software. When I run the import script for EBSD data, I got this message:

Warning: You have choosen to correct your EBSD data for differently aligned reference frames for the Euler angles and the map
coordinates. However, you have not specified which reference system setting has been used on your Edax system . I'm going to assume
"setting 1". Be careful, the default setting of EDAX is "setting 2". Click here for more information.Please make sure you have chosen
the correct setting and specify it explicitely using the syntax

ebsd = EBSD.load(fileName,'convertEuler2SpatialReferenceFrame','setting 2')

In loadEBSD_ang (line 219)
In EBSD.load (line 86)
In Ni17_0N_1 (line 29)

the file name is : Ni17_0N_1.ang
The path is D:\Users\mhatta\Documents\MATLAB\mtex-5.8.1\mtex-5.8.1\Ni17

I found the instruction on how to do the reference frame alignment and I follow these instructions:

ebsd = EBSD.load([mtexEBSDPath filesep 'Ni17_0N_1.ang'],'convertEuler2SpatialReferenceFrame','setting 2')
Error using assert
No file found!

Error in getFileNames (line 14)
assert(~isempty(files),'No file found!');

Could anyone please help me how I can handle this situation?

THank you before,
Fakhry

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1562>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INYM3ABFJF4FPQABDGTV6QY4ZANCNFSM6AAAAAAQOEEPOE>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1562", "url": "https://github.com/mtex-toolbox/mtex/discussions/1562", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
